packages/vuetify/src/components/VTooltip/_variables.scss

```diff
@@ -1,10 +1,10 @@
 @import '../../styles/styles.sass';
 
-$tooltip-background-color: rgba(map-get($grey, 'darken-2'), 0.9) !default;
-$tooltip-text-color: map-get($shades, 'white') !default;
+$tooltip-background-color: rgba(var(--v-theme-on-surface), 0.5) !default;
+$tooltip-text-color: rgb(var(--v-theme-surface)) !default;
 $tooltip-border-radius: $border-radius-root !default;
-$tooltip-font-size: 14px !default;
-$tooltip-transition-timing-function: map-get($transition, 'linear-out-slow-in') !default;
+$tooltip-font-size: .875rem !default;
+$tooltip-line-height: 1.6 !default;
```

Comment:
Any reasoning behind this **line-height** value?

Reply:
It was hard coded as font-size * 1.6